client/blocks/app-banner/style.scss

```diff
@@ -0,0 +1,68 @@
+.card.app-banner {
+	position: fixed;
+	bottom: 0;
+	width: 100%;
+	background-color: $blue-light;
+	color: $blue-dark;
+	/* Needed for banner to appear in front of notifications panel. */
+	z-index: z-index( 'root', '.app-banner' );
+
+	background-repeat: no-repeat;
+	/*
+	 * Needed to match the design as closely as possible,
+	 * we want the background image to be smaller than the containing div.
+	 */
+	background-size: 270px;
+	background-position: right 0 bottom 0;
+}
+
+.card.app-banner.post-editor {
+	background-image: url( '/calypso/images/illustrations/app-banner-editor.svg' );
+}
+
+.card.app-banner.notifications {
+	background-image: url( '/calypso/images/illustrations/app-banner-notifications.svg' );
+}
+
+.card.app-banner.reader {
+	background-image: url( '/calypso/images/illustrations/app-banner-reader.svg' );
+}
+
+.card.app-banner.stats {
+	background-image: url( '/calypso/images/illustrations/app-banner-stats.svg' );
+}
+
+.app-banner__text-content {
+	width: 70%;
```

Comment:
Good point, refactored to use `calc` in 05bedd4fe516cd20244fea026fa64804a19318b9.